bears/rest/reSTLintBear.py

```diff
@@ -8,7 +8,7 @@
 
 class reSTLintBear(LocalBear):
     LANGUAGES = {'reStructuredText'}
-    REQUIREMENTS = {PipRequirement('restructuredtext-lint', '1.0.0')}
+    REQUIREMENTS = {PipRequirement('restructuredtext-lint', '1.1.0')}
```

Comment:
the world has moved quite a bit since this issue was created.  Please also test with  https://github.com/twolfson/restructuredtext-lint/releases/tag/1.2.2

And since 1.0 was working, and you've tested 1.1, we shouldnt prevent people from using those version if they want to.

so the requirement should be `>1,<3`, or maybe just `~1.0` which will include all 1.x (and then we cross our fingers that they don't release a breaking change as a minor release.)

Reply:
Thanks @jayvdb :).
I'll make these changes.